### PR TITLE
Add food props batch 2

### PR DESCRIPTION
**Pull request by sumitnegi7**

### CODE REVIEW

File: src/data/food-compounds.ts [hunk 1]
```diff
@@ -1742,4 +1742,1340 @@ export const FoodCompounds: Record<string, FoodProps> = {
...
+    criticalCompressibility: 0.21,
+    accentricFactor: 1.34,
+    zRackett: 0.21,
+    prVolumeTransalationCoefficient: 0,
+    srkVolumeTransalationCoefficient: 0,
+    csAccentricFactor: 1.34,
+    csSolubilityParameter: 8.44,
+    csLiquidMolarVolume: 275.13,
+    igEnthalpyOfFormation25C: -1352.12,
+    igGibbsEnergyOfFormation25C: 4833.34,
```
Comment (maneetgoyal) [inline src/data/food-compounds.ts:1765]:
```suggestion
    igGibbsEnergyOfFormation25C: 483.34,
```

File: src/data/food-compounds.ts [hunk 2]
```diff
@@ -1742,4 +1742,1340 @@ export const FoodCompounds: Record<string, FoodProps> = {
...
+    vaporPressureConstantA: 0,
+    vaporPressureConstantB: 0,
+    vaporPressureConstantC: 0,
+    vaporPressureConstantD: 0,
+    vaporPressureConstantE: 0,
+    idealGasHeatCapacityEqnNumber: 4,
+    idealGasHeatCapacityConstantA: -133370,
+    idealGasHeatCapacityConstantB: 2373.82,
+    idealGasHeatCapacityConstantC: -2.2307,
+    idealGasHeatCapacityConstantD: 0.000852,
```
Comment (maneetgoyal) [inline src/data/food-compounds.ts:1776]:
```suggestion
    idealGasHeatCapacityConstantD: 0.00085236,
```

File: src/data/food-compounds.ts [hunk 3]
```diff
@@ -1742,4 +1742,1340 @@ export const FoodCompounds: Record<string, FoodProps> = {
...
+    criticalTemperature: 609.87,
+    criticalPressure: 2421879.84,
+    criticalVolume: 0.54,
+    criticalCompressibility: 0.26,
+    accentricFactor: 0.37,
+    zRackett: 0.26,
+    prVolumeTransalationCoefficient: 0,
+    srkVolumeTransalationCoefficient: 0,
+    csAccentricFactor: 0.37,
+    csSolubilityParameter: 0,
```
Comment (maneetgoyal) [inline src/data/food-compounds.ts:2030]:
```suggestion
    csSolubilityParameter: 6.13,
```

File: src/data/food-compounds.ts [hunk 4]
```diff
@@ -1742,4 +1742,1340 @@ export const FoodCompounds: Record<string, FoodProps> = {
...
+    csLiquidMolarVolume: 0,
```
Comment (maneetgoyal) [inline src/data/food-compounds.ts:2031]:
```suggestion
    csLiquidMolarVolume: 206.47,
```

File: src/data/food-compounds.ts [hunk 5]
```diff
@@ -1742,4 +1742,1340 @@ export const FoodCompounds: Record<string, FoodProps> = {
...
+    vaporPressureConstantEqnNumber: 0,
+    vaporPressureConstantA: 0,
+    vaporPressureConstantB: 0,
+    vaporPressureConstantC: 0,
+    vaporPressureConstantD: 0,
+    vaporPressureConstantE: 0,
+    idealGasHeatCapacityEqnNumber: 4,
+    idealGasHeatCapacityConstantA: -100010,
+    idealGasHeatCapacityConstantB: 1507.88,
+    idealGasHeatCapacityConstantC: -1.001,
```
Comment (maneetgoyal) [inline src/data/food-compounds.ts:2176]:
```suggestion
    idealGasHeatCapacityConstantC: -1.00156,
```

File: src/data/food-compounds.ts [hunk 6]
```diff
@@ -1742,4 +1742,1340 @@ export const FoodCompounds: Record<string, FoodProps> = {
...
+    vaporPressureConstantEqnNumber: 0,
+    vaporPressureConstantA: 0,
+    vaporPressureConstantB: 0,
+    vaporPressureConstantC: 0,
+    vaporPressureConstantD: 0,
+    vaporPressureConstantE: 0,
+    idealGasHeatCapacityEqnNumber: 4,
+    idealGasHeatCapacityConstantA: -136520,
+    idealGasHeatCapacityConstantB: 1715.06,
+    idealGasHeatCapacityConstantC: -1.05,
```
Comment (maneetgoyal) [inline src/data/food-compounds.ts:2309]:
```suggestion
    idealGasHeatCapacityConstantC: -1.30564,
```

File: src/data/food-compounds.ts [hunk 7]
```diff
@@ -1742,4 +1742,1340 @@ export const FoodCompounds: Record<string, FoodProps> = {
...
+    idealGasHeatCapacityConstantD: 0.0004,
```
Comment (maneetgoyal) [inline src/data/food-compounds.ts:2310]:
```suggestion
    idealGasHeatCapacityConstantD: 0.0004364,
```

File: src/data/food-compounds.ts [hunk 8]
```diff
@@ -1742,4 +1742,1340 @@ export const FoodCompounds: Record<string, FoodProps> = {
...
+    vaporPressureConstantEqnNumber: 0,
+    vaporPressureConstantA: 0,
+    vaporPressureConstantB: 0,
+    vaporPressureConstantC: 0,
+    vaporPressureConstantD: 0,
+    vaporPressureConstantE: 0,
+    idealGasHeatCapacityEqnNumber: 4,
+    idealGasHeatCapacityConstantA: -55357,
+    idealGasHeatCapacityConstantB: 1468.58,
+    idealGasHeatCapacityConstantC: -1.073,
```
Comment (maneetgoyal) [inline src/data/food-compounds.ts:2442]:
```suggestion
    idealGasHeatCapacityConstantC: -1.0734,
```

**Review by maneetgoyal — CHANGES_REQUESTED** [review]

**Comment by maneetgoyal** [comment]
Try to review the data 1 more time to ensure there are no other mistakes.

**Comment by sumitnegi7** [comment]
Sir @maneetgoyal  rechecked the values and updated the mistakes. Please have a look

**Review by maneetgoyal — APPROVED** [review]
LGTM

**Pull request merged**